tests/blockchain/eth/contracts/contracts/PolicyManagerTestSet.sol

```diff
@@ -86,15 +86,25 @@ contract StakingEscrowForPolicyMock {
 
     /**
     * @notice Emulate mint method
+    * @param _staker Staker on behalf of whom minting will be
     * @param _startPeriod Start period for minting
     * @param _numberOfPeriods Number periods for minting
     **/
-    function mint(uint16 _startPeriod, uint16 _numberOfPeriods) external {
+    function mint(address _staker, uint16 _startPeriod, uint16 _numberOfPeriods) public {
```

Comment:
Uh, maybe I missed something. Why not `msg.sender`?

Reply:
It's just for tests